features/support/web_server/two_headings_failures_and_one_warning.html

```diff
@@ -7,6 +7,6 @@
   <body>
     <p role="main">
     <!-- there is no content following the heading below -->
-    <h2>This is the main heading, but it's not an H1</h2>
+    <h4>This is the main heading, but it's not an H1 or H2</h4>
```

Comment:
Is this an example of no `<h1>` or no content following the heading? It would be good to ensure examples only fail for the reason they illustrate.